packages/miniapp-manifest-generator/package.json

```diff
@@ -39,7 +39,6 @@
     "globals": "^15.15.0",
     "jsdom": "^24.1.0",
     "tailwindcss": "^4.0.9",
-    "typescript": "~5.7.2",
```

Comment:
This was causing a ton of dependency resolution issues. We should only be using the root typescript version for packages in this repo.